targets/TARGET_STM/TARGET_STM32F0/TARGET_STM32F091xC/CMakeLists.txt

```diff
@@ -13,15 +11,19 @@ elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
     set(LINKER_FILE TOOLCHAIN_ARM/stm32f091xc.sct)
 endif()
 
-set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+add_library(STM32F091xC INTERFACE)
 
-target_sources(mbed-core
+target_sources(STM32F091xC
     INTERFACE
         system_clock.c
         ${STARTUP_FILE}
 )
 
-target_include_directories(mbed-core
+target_include_directories(STM32F091xC
```

Comment:
I assume this follows the naming convection we have in targets.json? do we need to keep this as mixed uppercase with lower case? 
I would use lower case in all targets if possible (the translation should be straightforward from old to new).

Reply:
This is the convention from the folder structure and targets.json. Generally CMake style is to use camel case for targets, which I see we don't follow in Mbed. Is the Mbed style to use all lowercase?